apps/cowswap-frontend/src/modules/swap/helpers/getAmountsForSignature.test.ts

```diff
@@ -119,7 +119,7 @@ describe('getAmountsForSignature()', () => {
     })
 
     /**
-     * Sell amount calculated as: inputAmount + slippage
+     * Sell amount calculated as: (inputAmount + fee) + slippage
```

Comment:
But we should not add fee when `fee=zero`, should we?
At least Haris said that:
https://cowservices.slack.com/archives/C036G0J90BU/p1703156515196189

Reply:
All good, discussed on the call